moz_minions/minions.py

```diff
@@ -49,16 +49,16 @@ def __init__(self, name, **kwargs):
         if 'serial' in kwargs:
             self.serial = kwargs['serial']
             os.environ['ANDROID_SERIAL'] = self.serial
-            self.description = ", Serial: " + self.serial
+            self.description += ", Serial: " + self.serial
         if 'command' in kwargs:
             self.command = kwargs['command']
-            self.description = ", Command: " + self.command
+            self.description += ", Command: " + self.command + "}"
         if 'output' in kwargs:
             self.output_file = kwargs['output']['file']
         self.kwargs = kwargs
 
```

Comment:
Do you want to assume that it will always have "name", "serial", and "command"?
If not:
1. remove "}" in line 55, and put it before "self.kwargs = kwargs"
2. check for None in  "name", "serial", and "command"

PS. using "is None" to test it should be faster than "== None"